fairlearn/reductions/_moments/weighted_error_rate.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+import numpy as np
+from itertools import repeat
+from .moment import LossMoment,ClassificationMoment,_ALL, _LABEL
+from .bounded_group_loss import SquareLoss
+_WEIGHTS = "weights"
+
+class WeightedErrorRate(LossMoment):
+    """Weighted Loss error."""
+    ## Not Sure about whether this should inherit from ClassificationMoment or LossMoment
```

Comment:
If it's for classification then ClassificationMoment, if it's for regression then LossMoment. If it could be either... then we have a new special case

Reply:
The calculation in signed_weights looks like binary classification so probably the former